django_mongodb/operations.py

```diff
@@ -49,6 +51,12 @@ def adapt_decimalfield_value(self, value, max_digits=None, decimal_places=None):
             return None
         return Decimal128(value)
 
+    def adapt_json_value(self, value, encoder):
```

Comment:
When is this method called and what's the purpose? I ask because `json.loads+json.dumps` is expensive and ends up with the same value. Also it won't work with pymongo's extended JSON types like ObjectId, Code, Binary, etc...

Reply:
Good point. We can skip it unless the user has specified a [custom `encoder`](https://docs.djangoproject.com/en/dev/ref/models/fields/#django.db.models.JSONField.encoder).